docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -74,3 +74,41 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
+
+## Alerting: Grafana OnCall integration
+
+<!-- George Robinson -->
+
+_Generally available in Grafana_
+
+Use the Grafana Alerting -  Grafana OnCall integration to effortlessly connect alerts generated by Grafana Alerting with Grafana OnCall, where you can then route them according to defined escalation chains and schedules.
+
+## Alerting: Insights (Cloud only)
```

Comment:
I'm going to move this over to the Cloud what's new since it's not an on-prem item. I'll ping you when I open that PR (probably tomorrow as I'm off this afternoon).